### ISSUE TITLE
Mislinked Charts

### ISSUE THREAD

**Issue by Scouter953** [Paturages/chorus#298]
On "This Will Be the Day," it says "Download PeckInkay and Harmonix's chart." However, both of the links are identical, so only PeckInkay's chart is actually available.

**Comment by Paturages** [comment]
Interesting take: links are all represented as orange in the site design, but you did interpret the 2 cyan colored charters as being 2 different links, which is an interesting user experience issue.

So while this may be a design issue, I'll just clarify that every line that says "Download x's chart" or "Download x and y's chart" will always only contain one same link. The "x and y" part, regardless of whether they're colored or not, says that either they worked together on that same chart, or perhaps one of them edited the other's chart. In any case, it means that both names have contributed to this chart.

In this particular case, PeckInkay chose to edit, from their own interpretation, the original chart from Harmonix. The link that is provided is only and solely PeckInkay's edit of the Harmonix chart, but it is undeniable that both Harmonix and PeckInkay have both contributed to the chart and should both be credited.

tl;dr: nah it's meant to be one link with 2 charters

**Comment by Scouter953** [comment]
Oh, I see. Yeah, it's a bit misleading that it breaks into two separate hyperlinks, then. Thanks for clearing that up though
